timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -217,6 +225,35 @@ types:
       LeaderTimes:
         fields:
           leaderTimes: map<Namespace, LeaderTime>
+      NamedMinTimestampLeaseRequest:
```

Comment:
The more I think about it the small ontology of this API is:

* The feature is called "Timestamp lease"
* You "acquire" a fresh "Timestamp lease" on a "TimestampLeaseName"
* You "get" "MinTimestampLeaseValue" ?

Reply:
Should I rename things following that?

Reply:
What do you think about this?

Reply:
Less words is better whenever possible, and the javadoc will cover what we are trying to do

The only thing that looked weird is "MinTimestampLeaseValue", not sure if the additional wrapping is informative

Reply:
`MinLeasedTimestamp` probably works for the last one (and generally I think this ontology is reasonable)